app/src/app/components/view-profile/view-profile.component.html

```diff
@@ -56,20 +56,28 @@ <h4 class="app-profile-general-dropdown-heading">{{'APP.EDIT_PROFILE.TELEPHONE'
           <p class="app-profile-general-dropdown-details" *ngIf="generalSectionButtonsToggle">{{personDetails?.telephone}}</p>
           <input type="tel" name="Telephone" value="" formControlName="telephone" *ngIf="!generalSectionButtonsToggle"
             class="app-profile-general-dropdown-input-details app-selected">
+          <mat-error class="app-error-message" *ngIf="profileForm.get('telephone').invalid  &&
+            (profileForm.get('telephone').dirty || profileForm.get('telephone').touched)">
+            {{ 'APP.EDIT_PROFILE.INVALID_INPUT' | translate }}
+          </mat-error>
         </div>
         <mat-divider class="app-general-dropdown-divider"></mat-divider>
```

Comment:
this PR does not customize the `app-general-dropdown-divider` style? is this class needed for this component?

Reply:
.app-general-dropdown-divider {
    border-top-color: var(--theme-grey-divider-color);
  }
Line 120